inputters/markdown.lua

```diff
@@ -79,10 +79,18 @@ local function SileAstWriter (options)
   writer.blockquote = simpleCommandWrapper("markdown:internal:blockquote")
   writer.verbatim = simpleCommandWrapper("verbatim")
   writer.listitem = simpleCommandWrapper("item")
-  writer.linebreak = simpleCommandWrapper("cr")
```

Comment:
Why is this removed? We'd still want to support the standard Markdown linebreaks.

Reply:
Mistake during testing thanks